WORKSPACE

```diff
@@ -701,6 +701,16 @@ http_archive(
     ],
 )
 
+
+# TODO(davido): This is not needed, because jdk compiler module patching is disabled for JDK 14
+http_archive(
```

Comment:
This is potentially using a different version of Java compiler. Remove it.

Reply:
Done.